src/client/pps/pps.proto

```diff
@@ -308,6 +321,7 @@ service API {
   rpc InspectJob(InspectJobRequest) returns (JobInfo) {}
   rpc ListJob(ListJobRequest) returns (JobInfos) {}
   rpc DeleteJob(DeleteJobRequest) returns (google.protobuf.Empty) {}
+  rpc RestartDatum(RestartDatumRequest) returns (google.protobuf.Empty) {}
```

Comment:
I'm actually not sure I understand how that makes adding fields easier. It seems like the major concern would be cross version compatibility but if I understand protobufs correctly then it'd be totally possible to deserialize a serialized `google.protobuf.Empty` as a `RestartDatumResponse` if they were both messages without any fields. So it seems like just introducing `RestartDatumResponse` the first time we want to add a field to it would work about as well.